examples/viewer.py

```diff
@@ -94,6 +97,40 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
         logger.setLevel("INFO")
         self.print_help_text()
 
+    def draw_contact_debug(self):
+        """
+        This method is called to render a debug line overlay displaying active contact points and normals.
+        Yellow lines show the contact distance along the normal and red lines show the contact normal at a fixed length.
+        """
+        yellow = mn.Color4(1.0, 1.0, 0.0, 1.0)
+        red = mn.Color4(1.0, 0.0, 0.0, 1.0)
+        cps = self.sim.get_physics_contact_points()
+        self.sim.get_debug_line_render().set_line_width(1.5)
+        camera_position = self.render_camera.render_camera.node.absolute_translation
+        # only showing active contacts
+        active_contacts = [x for x in cps if x.is_active]
```

Comment:
```suggestion
        active_contacts = (x for x in cps if x.is_active)
```
No reason it shouldn't be a generator.